app/Http/Controllers/ItemController.php

```diff
@@ -308,17 +312,14 @@ public function showWithGuild($guildSlug, $id, $slug = null)
                 ])
                 ->where('guilds.id', $guild->id);
             },
+            'priodCharacters' => function ($query) use ($guild) {

```

Comment:
I think I am missing something here.. You're querying all of the "priodCharacters", but only filtering on the guild id. Just trying to understand what you're doing here and the intent.

Reply:
Ignore me.. IIRC this is querying the item, getting all characters with that prioed then filtering by the guildID. Got it.

Reply:
Correct.

Reply:
One thing to watch out for with Laravel is that if you don't intentionally eager load a relationship, and you make a call to that relationship, it will make the call for you. I'm very certain this catches a lot of devs off guard, and they end up with an exceptionally high number of database queries without realizing it.

Imagine not eager loading, then calling different mutations of `$item->priodCharacters` in a for loop. Ouch.

Reply:
Yeah that would be bad lol.